CONTRIBUTING.md

```diff
@@ -35,3 +38,12 @@ When landing large or substantial changes, we want to make sure users are aware
 - All new features
 - All API changes
 - Large bug fixes
+
+### Where to contribute
+Some areas that are best suited for external contributions are:
+- Adding SQL functions
+- Adding pg_catalog tables to support more [tools and integrations](https://materialize.com/docs/integrations/)
+
+It's best to ask on the Github issue before contributing to these areas:
```

Comment:
```suggestion
Areas that are not well suited for external contributions:
```

Reply:
I can't imagine @danhhz will object to this, but tagging him for explicit confirmation anyway.